Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -134,6 +134,20 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Toggles displaying alternative routes when in active navigation session.
```

Comment:
This makes it sound like this property only works when NavigationMapView is embedded inside a NavigationViewController with a Router hooked up to it, which would strongly suggest moving the property to NavigationViewController (see also #3473). However, it looks like this functionality only depends on the `continuousAlternatives` property being set. This documentation should clarify what’s actually required, so that custom UI implementers have a sense of whether it’ll work for them.